doc/modules/cross_validation.rst

```diff
@@ -468,38 +468,38 @@ validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
 
-Label-Shuffle-Split
+Group-Shuffle-Split
```

Comment:
I wonder: why are we styling these headings with hyphens, why not "Group Shuffle Split"? or similarly "Leave One Out"?